log/scope_test.go

```diff
@@ -218,6 +220,130 @@ func TestBasicScopes(t *testing.T) {
 	}
 }
 
+func TestScopeAddLabel(t *testing.T) {
+	const name = "TestScope"
+	const desc = "Desc"
+	s := RegisterScope(name, desc, 0)
+	s.SetOutputLevel(DebugLevel)
+
+	lines, err := captureStdout(func() {
+		funcs.Store(funcs.Load().(patchTable))
+		s.AddLabel("foo", "bar").AddLabel("baz", 123).AddLabel("qux", 0.123).Debuga("Hello")
+		s.Debuga("Hello")
+		_ = Sync()
+	})
+
+	if err != nil {
+		t.Errorf("Got error '%v', expected success", err)
+	}
+
+	expect := `Hello {"foo":"bar","baz":1,"qux":0.123}`
+	mustRegexMatchString(t, lines[0], expect)
+	mustRegexMatchString(t, lines[1], expect)
```

Comment:
why is there 2 lines?
Why is it a regex match instead of exact match - from the timestamp?